pip-dep/requirements.txt

```diff
@@ -9,6 +9,7 @@ requests~=2.22.0
 scipy~=1.4.1
 syft-proto~=0.2.9.a1
 tblib~=1.6.0
+git+https://github.com/OpenMined/TenSEAL.git#egg=tenseal
```

Comment:
If we're going to add a dependency, we should probably do it via a versioned package.

Reply:
We have some packaging issues on the TenSEAL side, that we can't deal with for the moment, is it impossible to do it this way?

Reply:
Well, I'm not saying we absolutely can't, but I'd generally discourage it since it can make PySyft brittle when changes happen upstream. What are the upstream packaging issues? TenSEAL is an OpenMined library, so maybe we should iron those out and get a versioned release published.

Reply:
TenSEAL has a binary extension that need to be packaged, and we should automate the build and packaging for different platforms, and we have some other priorities ATM. I'm working on TenSEAL and will be around if any changes comes in. We should just consider it a temporary thing until we get it published.

Reply:
Hmm - i think this temporary thing is a big enough detraction from packaging standards to warrant holding it out of master.  Happy to push things in a branch for now though until it's ready.

Reply:
Alternately: make it an optional dependency, and pin the dependency to a specific commit. The issue with brittleness comes from optimistically assuming that future versions will continue to be compatible (like PySyft used to do.)

Reply:
Just pushed a package that should work on most linux distro, next is to automate the build for other platforms.